torntools/scripts/content/jail/ttJail.js

```diff
@@ -66,6 +66,8 @@ function addFilterToTable(list, title) {
 
 		filter_container.find("#tt-faction-filter").appendChild(option);
 	}
+	let ownFacOption = doc.new({ type: "option", value: userdata.faction.faction_tag, text: userdata.faction.faction_tag});
```

Comment:
Own faction should be above the manual factions imo

Reply:
Done.

Reply:
You misunderstood, you put it even lower instead of higher.

Reply:
At the last of the list of factions gathered ?